pkg/apis/migration/v1alpha1/migtoken_types.go

```diff
@@ -306,9 +306,6 @@ func (r *MigToken) GetTokenReview(client k8sclient.Client) (*v1beta1.TokenReview
 	if err != nil {
 		return nil, err
 	}
-	if cluster == nil {
```

Comment:
+1

Reply:
actually -1, need to rebase, so this doesnt get dropped

Reply:
Woops.  Done.